client/dashboard/emails/index.tsx

```diff
@@ -28,7 +32,23 @@ function Emails() {
 	const { data: filteredData, paginationInfo } = filterSortAndPaginate( emails, view, emailFields );
 
 	return (
-		<PageLayout header={ <PageHeader /> } notices={ <OptInWelcome tracksContext="emails" /> }>
+		<PageLayout
+			header={
+				<PageHeader
+					actions={
+						<>
+							<Button className="emails__add-email-forwarder" variant="link" __next40pxDefaultSize>
```

Comment:
We should use `variant="secondary"` for the consistency.

Reply:
We should ask designers to see whether it makes sense. I'm not sure why it should be a link 🤔